### DISCUSSION TITLE
Latest wikijs version sensitive SSO and GitHub data

### DISCUSSION THREAD

**Discussion by InSelfControll** [requarks/wiki#7277] [Error / Bug Report] [answered]
I'm using the latest wikijs version in docker and I saw that the private key for ssh login to GitHub.

I have added GitHub and discord authentication to my wikijs and I noticed it doesn't hide the client secret it's all plain text.

Same goes for GitHub ssh login for push, pull and backup the MD files from my wiki.

Why it shows these details in plain text and not hiding them / encrypting them?

**Comment by NGPixel** [comment, marked as answer]
It's only shown to the root administrator group. This group has full access and will therefore see these values. 

Don't put users into that group if you don't want them to see sensitive values.
